output/elasticsearch/non_indexable_policy.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package elasticsearch
+
+/*
+const (
```

Comment:
I think this entire file can probably be deleted, the dead letter index was never taken out of beta in beats and isn't supported under agent.